community/psmisc/peekfd_lintl.patch

```diff
@@ -0,0 +1,30 @@
+Author: Breno Leitao <brenohl@br.ibm.com>
+Date:   Thu Apr 6 14:03:00 2017 -0300
+
+    pstree: Linking with libintl.
+    
+    Curently pstree does built, at least on ppc64le, because it does
+    not have a explicit link against gettext, causing the following error:
+    
+      peekfd.o: In function `print_version':
+      peekfd.c:(.text+0x160): undefined reference to `libintl_gettext'
+      peekfd.c:(.text+0x18c): undefined reference to `libintl_gettext'
+      peekfd.c:(.text+0x1ac): undefined reference to `libintl_gettext'
+    
+    peekfd depends on libint because of the following macro:
+    
+      #define _(String) gettext (String)
```

Comment:
Why not just make _() macro a noop?

We generally do not want to internationalize CLI tools in Alpine, only GUI software.